### PR TITLE
[CHORE] Update libraries

### PR DESCRIPTION
**Pull request by filipedelimabrito**
This PR updates some libraries used in this project.

It also changes from single quotes to double ones. Although both are acceptable, it is a good idea to use double quotes, especially when you are already using string interpolation.

### CODE REVIEW

File: livedata/build.gradle [hunk 1]
```diff
@@ -80,65 +78,64 @@ android {
...
     // optional - Kotlin Extensions and Coroutines support for Room
     implementation "androidx.room:room-ktx:$room_version"
-    implementation 'com.google.code.gson:gson:2.8.6'
+    implementation "com.google.code.gson:gson:2.8.6"
     implementation "androidx.room:room-ktx:$room_version"
-    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.5'
-    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.5'
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.8"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.6"
     // Kotlin + coroutines
```
Comment (filipedelimabrito) [inline livedata/build.gradle:95]: Is this comment useful?

Comment (JcMinarro) [reply livedata/build.gradle:95]: No, it isn't

File: livedata/build.gradle [hunk 2]
```diff
@@ -23,17 +23,15 @@ jacocoAndroidUnitTestReport {
...
 }
 
-
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
     buildToolsVersion "29.0.3"
     defaultConfig {
         minSdkVersion 21
-        targetSdkVersion 28
```
Comment (JcMinarro) [inline livedata/build.gradle:36]: The SDK is not ready to be targeted to Android SDK Version 29 yet, we need to do some changes previously

File: build.gradle [hunk 3]
```diff
@@ -3,27 +3,24 @@
 buildscript {
     ext {
         kotlin_version = '1.3.72'
-        jacocoVersion = '0.1.4'
-        junitVersion = '4.12'
-        jupiter_version = '5.5.1'
-        mannodermausVersion = '1.5.1.0'
+        mannodermaus_version = '1.5.1.0'
```
Comment (filipedelimabrito) [inline build.gradle:6]: Removed unused extra properties. Also changes from `mannodermausVersion` to `mannodermaus_version ` - always a good idea to have the same pattern (I could see something_version is being most used here, so the way to go now).

File: livedata/build.gradle (hunk 2 above)
Comment (filipedelimabrito) [reply livedata/build.gradle]: @JcMinarro, it is now addressed by putting the version 28 again.
Thank you for reviewing.

30c5fa4

File: build.gradle (hunk 3 above)
Comment (JcMinarro) [reply build.gradle:6]: Yes, it is fine for now. We are migrating it to a Kotlin file into the `buildSrc` project.

**Pull request merged**